Makefile

```diff
@@ -309,19 +309,19 @@ ifeq ($(filter clean distclean print-%,$(MAKECMDGOALS)),)
     ifeq ($(DUMMY),FAIL)
       $(error Failed to extract assets from US ROM)
     endif
-    ifneq (,$(wildcard baserom.jp.z64))
+    ifneq (,$(shell python3 tools/detect_baseroms.py jp))
       DUMMY != $(PYTHON) extract_assets.py jp >&2 || echo FAIL
       ifeq ($(DUMMY),FAIL)
         $(error Failed to extract assets from JP ROM)
       endif
     endif
-    ifneq (,$(wildcard baserom.eu.z64))
+    ifneq (,$(shell python3 tools/detect_baseroms.py eu))
       DUMMY != $(PYTHON) extract_assets.py eu >&2 || echo FAIL
       ifeq ($(DUMMY),FAIL)
         $(error Failed to extract assets from EU ROM)
       endif
     endif
-    ifneq (,$(wildcard baserom.sh.z64))
```

Comment:
Doesn't probably matter that much, but why aren't we forcing the top version to be US always here?

Reply:
good point, addressed